.circleci/build.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -ex
+
+export MAX_JOBS=8
+
+# setup sccache wrappers
+if hash sccache 2>/dev/null; then
+    SCCACHE_BIN_DIR="/tmp/sccache"
+    mkdir -p "$SCCACHE_BIN_DIR"
+    for compiler in cc c++ gcc g++ x86_64-linux-gnu-gcc; do
+        (
+            echo "#!/bin/sh"
+            echo "exec $(which sccache) $(which $compiler) \"\$@\""
+        ) > "$SCCACHE_BIN_DIR/$compiler"
+        chmod +x "$SCCACHE_BIN_DIR/$compiler"
+    done
+    export PATH="$SCCACHE_BIN_DIR:$PATH"
+fi
+
+GLOW_DIR=$PWD
+
+# Install llvm
+sudo apt-get update
+sudo apt-get install -y llvm-6.0 llvm-6.0-dev libpng-dev
+
+# Install ninja and (newest version of) cmake through pip
+sudo pip install ninja cmake
+hash cmake ninja
+
+# Build glow
+cd ${GLOW_DIR}
+mkdir build && cd build
+CMAKE_ARGS=()
+CMAKE_ARGS+=("-DGLOW_WITH_OPENCL=OFF")
+CMAKE_ARGS+=("-DGLOW_WITH_CPU=ON")
+if [[ "$CIRCLE_JOB" == asan ]]; then
+    CMAKE_ARGS+=("-DGLOW_USE_SANITIZER='Address;Undefined'")
+fi
+if [[ "$CIRCLE_JOB" == debug ]]; then
+    CMAKE_ARGS+=("-DCMAKE_BUILD_TYPE=Debug")
+else
+    CMAKE_ARGS+=("-DCMAKE_BUILD_TYPE=Release")
+fi
+cmake -GNinja ${CMAKE_ARGS[*]} ../
```

Comment:
where is the compiler set? We'd need to set it to clang and clang++.

Reply:
The by-default compiler in this environment is clang I think

Reply:
Yup
```
-- The C compiler identification is Clang 6.0.1
-- The CXX compiler identification is Clang 6.0.1
```